worker/worker.py

```diff
@@ -529,7 +521,7 @@ def setup_fastchess(worker_dir, compiler, concurrency, global_cache, tests=False
                     "Executing {} failed. Error: {}".format(cmd, errors)
                 )
 
-        shutil.copy("fastchess" + EXE_SUFFIX, testing_dir)
+        (build_dir / "fastchess").with_suffix(EXE_SUFFIX).rename(fastchess_path)
```

Comment:
I think this broke something..

yesterday fauzi got an error after a worker update which required a new fastchess build

![grafik](https://github.com/user-attachments/assets/a8cb449a-a57f-4e0c-97c6-5cec8195d9c7)

"Winerror cannot create a file when that file already exists"

Reply:
This night I will trigger the fastchess rebuild to check if I'm able to replicate the issue.

Reply:
fixed in #2248